src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -332,6 +332,7 @@ def prepare_or_run_pipeline(
                     else None
                 ),
             )
+            pipeline_tags = step_settings.pipeline_tags
```

Comment:
Ah another thing: This is taking the pipeline tags from the settings specified for a specific step? I'm guessing this is in a for-loop and will just use the settings of the latest step.

Instead, I think we should use the tags defined on the pipeline settings (`self.get_settings(deployment)`)

Reply:
pfft right again. Im rusty AF. Sorry